ansible/roles/biocache-service/tasks/main.yml

```diff
@@ -68,36 +68,6 @@
         use_cache: true
         force_cache: true
         proxy_pass: "http://127.0.0.1:8080{{biocache_service_proxy_target}}/occurrence/legend"
-      - path: "{{biocache_service_context_path}}/occurrence/facets"
```

Comment:
These removals need to be replaced with a configurable approach. The ALA rely heavily on these rules to cater for the amount of traffic that we serve on our low budget hardware.

Reply:
Right, I've miss them. They are not part of my modification.
I'll be back with an update.